src/components/common/Button/button.module.css

```diff
@@ -0,0 +1,37 @@
+.button {
+  color: orange;
+  display: block;
+  width: 100%;
+  padding: 0 16px;
+  height: 40px;
+  text-transform: uppercase;
+  font-size: 14px;
+  transition: color 0.2s ease, background-color 0.2s ease, outline 0.2s ease;
+  border-radius: 2px;
+  min-width: 120px;
+}
+.button:hover {
+  color: var(--color-orange-light);
```

Comment:
we don't have these colors 
- --color-orange-light
- --color-orange-dark
- --color-black
- --color-white